lms/djangoapps/courseware/module_render.py

```diff
@@ -518,6 +519,9 @@ def rebind_noauth_module_to_user(module, real_user):
     # to the Fragment content coming out of the xblocks that are about to be rendered.
     block_wrappers = []
 
+    if settings.FEATURES.get("LICENSING", False):
+        block_wrappers.append(wrap_with_license)
```

Comment:
It would be good to have an equivalent of this in the Studio preview wrappers, so that an author can see the licenses that have been specified.